iset.mm

```diff
@@ -183325,6 +183374,232 @@ mathematician Ptolemy (Claudius Ptolemaeus).  This particular version is
+      CUWBVHAUCUDVVCCSVXQVXTWUSAWUTVHZUWCAVYHVXGIWGVYHAVXGQVYHWUCWPVNXSACVULCTM
+      ZLMZVULUNMZVVCWVKVULUNMZLMVYBWVHACWVKVULVWRAVULCWUMVWRYEWUMWUNUWDAWVMVUKC
+      TMZVULUNMZVYBAWVLWVOVULUNAWVLCWVOCUGMZLMWVOAWVKWVQCLAWVKWVOKCTMZUGMWVQAVU
+      KKCAVUKVWHVNZAUWEZVWRUWFAWVRCWVOUGACVWRUWHYFYGYFACWVOVWRAVUKCWVSVWRYEUWIY
+      GYHAVUKCVULWVSVWRWUMWUNUWJZYGAWVNCVVCLACVULVWRWUMWUNYIYFYJUWKZWNVYFVUOUTI
+      ZWWCWVCVUONVYFVUOWUDVNZWWDVYHVUOHVUOUTHUAXKXTYAWFYBYGAVYMVYBNZVXEVYEAVYBK
+      CYKMZVYLVYHKUWQMZHWLZVYMAWVPVULWWFTMZVULUNMVYBWWFAWVOWWIVULUNAWVOJJBUFMZT
+      MZCTMJWWJCTMZTMZWWIAVUKWWKCTAVUKWWJJTMZWWKAVVQBYLIZVUKWWNNVAABDWQZJBUWLVM
+      AWWJUTIVVQWWNWWKNAWWJAJQIWWOWWJQIZUWMWWPJBUWNVMZVNZVAWWJJUWOYMYGYHAJWWJCA
+      UWRWWSVWRUWPAKWWLYKMZKWWJYKMZWWFTMZWWMWWIAVWEWWQVVBWWJCYNMKNZWWTWXBNWVTWW
+      REAJCYNMKNZWXCAJCOPWJZWXDFAJUEIVXSWXEWXDVSUWSVXTJCUWTVMVQJSIAVXSWWOWXDWXC
+      ULUXHVXTWWPJCBUXAWAYOKWWJCUXBUXCGAWXAVULWWFTAKJVUJKUGMZUFMZYKMZWXAVULAWXG
+      WWJKYKAWXFBJUFAVWFVWEWXFBNVWGVOBKUXDYMYFYFAVWPWXHVULNVWAVUJUXEVTUXFYHYJUX
+      GYHWWAAWWFVULAWWFAKYLIVVBWWFQIUXIEKCUXJVMZVNWUMWUNYIYJZAVWEVVBWWFWWHNVOEK
+      CHUBUXKVMAVYLWWGVYHHAVYPWGZVYHWXKVYHWXKVYLQVYHVYRAVYPXPUXRZVLUXLUXMUXNZWN
+      YPVYFVYBVYCRPZVYDWJZVYFVYBVUOAVYBVFIVXEVYEAVUKVVCVWIVVJUXOZWNVYFVUOWUDVLU
+      XPVYFVYBSIZVYCSIZWXNWXOVSAWXQVXEVYEAVYBAVYBWWFQWXJWXIUXQZVJZWNVXFWXRVYEVX
+      FVYCVXFVYBVUOAVYBQIVXEWXSWDVXOUXSVJWDVYBVYCYQWFVQUXTUYAVUOVVCCUYBVTUYCUYD
+      ZAVUTVXCUAQAVUSVXBVUPAVUQVXAVURAKVVCVUOAKVVCNZKCNZAKCACKAKCVVDVWTYDYRZUYH
+      AKCOPZWYBWYCUKZAVXSWYEVXTCUYEVTZAVXCWYEWYFULUAQKVUQVUPWYEVXBWYFVUOKCOXDVU
+      QVXAWYBVURWYCVUOKVVCYSVUOKCYSUYFUYGWYAKQIZAYTVHUYIYOUYJUYKUYLUYMUYNWHUACU
+      YRWBAVUMWCZVYBKLMZVYBWIPZWJZVUMAVXSWUJWYIVXTWULCVULWEWFAVYBWYJRPZWYLAVYBW
+      XPUYOAWXQWYJSIWYMWYLVSWXTAVYBWXTUYSVYBWYJYQWFVQAWYKCVULACVULYCZWYKULWYIAW
+      YNWYKAWYNWGZVVCCKWKZVYHHWLZVYMWYJVYBWIAWYQVYMWIPWYNAVYLVYHWYPHVYOWXKVYHWX
+      LURWXKVYHWXKVYHWXLWQWRAVYKUBQWYPAWYPVXGKWSZXAZQVVCCKWTZAVXGWYRQWUCWYHWYRQ
+      WOAYTKQUYPUYTXBXCZAWUFWUGVYJKNZXLVYKVYJWYPIAWUFVYKWUGXUBWUOWUQAVYKXUBWYEW
+      YGVYJKCOXDXEXFVYJVVCCKXGXHXMAUCUDVVCCKSVXQVXTVVTWVJXIZXJWDWYOWYQWVBWYRVYH
+      HWLZLMZWYJWYOVXGWYRVYHWYPHWYOKVXGIWJVXGWYRXNXONWYOKVVCCWYOVVCKAVVCKYCWYNA
+      VVCKCVULACVULVWRWUMWUNUYQVUAVUBYRAKCYCWYNWYDWDVUCVXGKXQXRWYPWYSNWYOWYTVHA
+      WYPWMIWYNXUCWDWYOVYHWYPIWGVYHWYOWYPQVYHAWYPQWOWYNXUAWDWPVNXSAXUEWYJNWYNAW
+      VBVYBXUDKLWWBAVWDVWEXUDKNVVDVOVYHKHKVFVYHKNXTYAYMYBWDYGAWWEWYNWXMWDYPVUDV
+      UEVUFVUGVUH $.
+  $}
+
+  ${
+    $d p N $.
+    $( The Euclid-Euler theorem, or Perfect Number theorem.  A positive even
+       integer ` N ` is a perfect number (that is, its divisor sum is ` 2 N ` )
+       if and only if it is of the form ` 2 ^ ( p - 1 ) x. ( 2 ^ p - 1 ) ` ,
+       where ` 2 ^ p - 1 ` is prime (a Mersenne prime).  (It follows from this
+       that ` p ` is also prime.)  This is Metamath 100 proof #70.
```

Comment:
Reference to ~mersenne should be added, maybe:  "where ` 2 ^ p - 1 ` is prime (a Mersenne prime), and therefore ` p ` is also prime, see ~ mersenne ."